src/backend/marsha/core/api.py

```diff
@@ -413,10 +413,10 @@ def initiate_live(
         key = f"{video.pk}_{stamp}"
 
         live_info = create_live_stream(key)
-        live_info.update({"type": serializer.validated_data["type"]})
```

Comment:
should you plan a data migration to fill the new field instead of this one for existing videos?

Reply:
It depends if we had a default value for the `live_type` property. A live is ephemeral so I think we don't need a data migration. But if we decided to add a default value, then we should create a data migration.

Reply:
A data migration is added.